wasp/apps/pager.py

```diff
@@ -82,19 +82,53 @@ class NotificationApp(PagerApp):
 
     def __init__(self):
         super().__init__('')
+        self.confirmation_view = wasp.widgets.ConfirmationView()
+        self.is_in_confirmation_view = False
 
     def foreground(self):
         notes = wasp.system.notifications
-
         id = next(iter(notes))
         note = notes[id]
         del notes[id]
         title = note['title'] if 'title' in note else 'Untitled'
         body = note['body'] if 'body' in note else ''
         self._msg = '{}\n\n{}'.format(title, body)
 
+        wasp.system.request_event(
+            wasp.EventMask.TOUCH
+            | wasp.EventMask.SWIPE_LEFTRIGHT
+        )
         super().foreground()
 
```

Comment:
To code to deactivate the confirmation view in the background() method seems to be missing (it means physical button and display timeout will not be handled correctly).